src/containerapp/azext_containerapp/custom.py

```diff
@@ -2637,3 +2641,240 @@ def list_environment_telemetry_otlp(cmd,
+                        target_port=None,
+                        registry_server=None,
+                        registry_pass=None,
+                        registry_user=None):
+    raw_parameters = locals()
+    session_pool_decorator = SessionPoolUpdateDecorator(
+        cmd=cmd,
+        client=SessionPoolPreviewClient,
+        raw_parameters=raw_parameters,
+        models=CONTAINER_APPS_SDK_MODELS
+    )
+    session_pool_decorator.construct_payload()
+    r = session_pool_decorator.update()
+
+    return r
+
+
+def show_session_pool(cmd,
+                      name,
+                      resource_group_name):
+    raw_parameters = locals()
+    session_pool_decorator = SessionPoolPreviewDecorator(
+        cmd=cmd,
+        client=SessionPoolPreviewClient,
+        raw_parameters=raw_parameters,
+        models=CONTAINER_APPS_SDK_MODELS
+    )
+    session_pool_decorator.validate_subscription_registered(CONTAINER_APPS_RP)
+    r = session_pool_decorator.show()
+
+    return r
+
+
+def list_session_pool(cmd,
+                      resource_group_name=None):
+    raw_parameters = locals()
+    session_pool_decorator = SessionPoolPreviewDecorator(
+        cmd=cmd,
+        client=SessionPoolPreviewClient,
+        raw_parameters=raw_parameters,
+        models=CONTAINER_APPS_SDK_MODELS
+    )
+    session_pool_decorator.validate_subscription_registered(CONTAINER_APPS_RP)
+    r = session_pool_decorator.list()
+
+    return r
+
+
+def delete_session_pool(cmd,
+                        name,
+                        resource_group_name):
+    raw_parameters = locals()
+    session_pool_decorator = SessionPoolPreviewDecorator(
+        cmd=cmd,
+        client=SessionPoolPreviewClient,
+        raw_parameters=raw_parameters,
+        models=CONTAINER_APPS_SDK_MODELS
+    )
+    r = session_pool_decorator.delete()
+
+    return r
+
+# session code interpreter commands
+def execute_session_code_interpreter(cmd,
+                        name,
+                        resource_group_name,
+                        identifier, 
+                        code,
+                        session_pool_location=None,
+                        timeout_in_seconds=None,
+                        ):
+    raw_parameters = locals()
+    session_code_interpreter_decorater = SessionCodeInterpreterCommandsPreviewDecorator(
+        cmd=cmd,
+        client=SessionCodeInterpreterPreviewClient,
+        raw_parameters=raw_parameters,
+        models=CONTAINER_APPS_SDK_MODELS
+    )
+    session_code_interpreter_decorater.validate_arguments()
+    session_code_interpreter_decorater.register_provider(CONTAINER_APPS_RP)
+
+    session_code_interpreter_decorater.construct_payload()
+    r = session_code_interpreter_decorater.execute()
+
+    return r
+
+def upload_session_code_interpreter(cmd,
+                        name,
+                        resource_group_name,
+                        identifier, 
+                        filepath
+                        ):
```

Comment:
PEP 8: E128 continuation line under-indented for visual indent
![image](https://github.com/Azure/azure-cli-extensions/assets/46143499/ae0098bd-46ec-4327-b203-a9fa1fe55e77)

Reply:
fixed :)